global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "msbuild-sdks": {
+        "Microsoft.Build.NoTargets": "1.0.80"
```

Comment:
Can we put the version in the bootstrap file itself?

Reply:
Removed global.json and added directly to build\build.proj